distribution/docs/src/main/resources/content/_running/command-scheduler.adoc

```diff
@@ -69,7 +67,7 @@ For example, if an `echo "Hello World"` command is set to run every five seconds
 
 In short, administrators can view the status of a run within the log as long as INFO was set as the status level.
 
-===== CQL Syntax
+=== CQL Syntax
```

Comment:
This section should no be under `Command Scheduler`. Is would fit better under `Console Commands`.

Reply:
Moved.